R/contingencytables.R

```diff
@@ -202,39 +202,42 @@ ContingencyTables <- function(jaspResults, dataset, options, ...) {
   }
 }
 
-.crossTabLogOdds <- function(jaspResults, dataset, options, analyses, ready) {
-  if(!options$oddsRatio)
-    return()
 
-  for (i in 1:nrow(analyses)){
+.crossTabOdds <- function(jaspResults, dataset, options, analyses, ready){
+  if (!options$oddsRatioNormal)
+    return ()
+  
+  for (i in 1:nrow(analyses)) {
     analysis <- analyses[i,]
-    analysisContainer <- jaspResults[[paste0("container", i)]]
-    if (!is.null(analysisContainer[["crossTabLogOdds"]]))
+    analysisContainer <- jaspResults[[paste0("container",i)]]
+    if (!is.null(analysisContainer[["crossTabOdds"]]))
```

Comment:
so here you change the name of the container, but you did not change the same name in the unit test file. That's probably why the unit tests failed. Either adjust that file to use the same name or revert the name change.